packages/payment-detection/src/erc20/transferrable-receivable.ts

```diff
@@ -0,0 +1,110 @@
+import { ExtensionTypes, PaymentTypes, RequestLogicTypes } from '@requestnetwork/types';
```

Comment:
I wish there was a `test/erc20/transferable-receivable.test.ts` to go along with this file.

Some test cases I'd like to see:

- ERC20TransferableReceivablePaymentDetector can retrieve payment using thegraph info retriever
- ERC20TransferableReceivablePaymentDetector can retrieve payment using proxy info retriever
- ERC20TransferableReceivablePaymentDetector ignores payments that bypass the TransferableReceivable (that don't call payOwner)